### PR TITLE
Add talks listing and pages

### PR DESCRIPTION
**Pull request by tylerdave**
Initial attempt at adding talk listings

### CODE REVIEW

File: src/pages/events/talks.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          <meta property="og:title" content={pageTitle} />
+        </Helmet>
+        <h1>{pageTitle}</h1>
+
+        {talkList.map(({ node: talk }) => (
+          <div key={talk.id}>
+            <h2>
+              <Link
+                to={`/events/talks/${talk.slug}`}
+                title="Presentation details"
```
Comment (Kataract) [inline src/pages/events/talks.js:26]: don't need this

Comment (tylerdave) [reply src/pages/events/talks.js:26]: do you specifically mean the link title? or am I missing something else?

Comment (Kataract) [reply src/pages/events/talks.js:26]: correct, the title attribute isn't needed

**Comment by tylerdave** [comment]
Merging this as good enough for now 😬

**Pull request merged**